acceptance/tests/restart_host_run_puppet.rb

```diff
@@ -9,9 +9,9 @@
     skip_test('QENG-3629 - AIX hosts cannot be restarted')
   end
 
-  applicable_agents = applicable_agents.select { |agent| agent['platform'] !~ /cisco_nexus/}
+  applicable_agents = applicable_agents.select { |agent| agent['platform'] !~ /cisco_nexus/ && agent['platform' !~ /cisco_ios_xr/ }
```

Comment:
This line is missing a closing ]
Could this just be

    { |agent| agent['platform'] !~ /cisco_nexus|cisco_ios_xr/ }

?